libs/data/src/exporter.ts

```diff
@@ -101,17 +101,22 @@ export class Exporter {
 
     await mkdir(pathParts.dir, { recursive: true });
 
-    return ok(
-      await splitToFiles(
-        typeof data === 'string' ? Readable.from(data) : data,
-        {
-          size: maximumFileSize ?? Infinity,
-          nextPath: (index) =>
-            join(pathParts.dir, `${pathParts.base}-part-${index + 1}`),
-          singleFileName: pathParts.base,
-        }
-      )
+    const paths = await splitToFiles(
+      typeof data === 'string' ? Readable.from(data) : data,
+      {
+        size: maximumFileSize ?? Infinity,
+        nextPath: (index) =>
+          join(pathParts.dir, `${pathParts.base}-part-${index + 1}`),
+        singleFileName: pathParts.base,
+      }
     );
+    // If the data was empty, splitToFiles won't create any files, but we still
+    // want to create an empty file.
+    if (paths.length === 0) {
+      writeFileSync(safePath, '');
```

Comment:
This is an `async` function. Why not this?

```suggestion
      await writeFile(safePath, '');
```

Reply:
Sure, can do. I've been switching back and forth so much between diff files that have diff version of `fs` I just lost track